### ISSUE TITLE
[Bug?]A10/SU35 battle area/zone same with ground vehicles.

### ISSUE THREAD

**Issue by Target5150** [KrazyIvan777/CQ-MoreVehicles#1]
Title. I'm not sure it was bug or because official A10/SU35 only in Rush mode so CQ/CQL didn't setting Jet's area.
![A10](https://user-images.githubusercontent.com/42076548/144589936-c1dd460f-362b-4f01-90ae-863570cc5504.png)
![F18](https://user-images.githubusercontent.com/42076548/144589942-f3ef1a32-1055-4ff6-ac60-34855a91e15b.png)

A10 in RUSH.
![RUSH A10](https://user-images.githubusercontent.com/42076548/144590719-b393d713-0397-4694-8a23-10e1aba88ea5.png)

**Comment by KrazyIvan777** [comment]
Yeah I know, unfortunately I don't know how to do that at the moment. If you can help me I would greatly appreciate it. I will find a good for this hopefully in a future update. If you want to disable out of bounds, you can add "Vu.desertionallowed true" in startup.txt for a workaround
